src/saga/adaptors/lsf/lsfjob.py

```diff
@@ -187,22 +187,40 @@ def _lsfcript_generator(url, logger, jd, ppn, lsf_version, queue, span):
     elif (jd.queue is None) and (queue is not None):
         lsf_params += "#BSUB -q %s \n" % queue
 
-    if jd.project is not None:
+    if jd.project is not None and ':' in jd.project:
+        account, reservation = jd.project.split(':',1)
+        lsf_params += "#BSUB -P %s \n" % str(account)
+        lsf_params += "#BSUB -U %s \n" % str(reservation)
+    elif jd.project is not None:
         lsf_params += "#BSUB -P %s \n" % str(jd.project)
+
     if jd.job_contact is not None:
         lsf_params += "#BSUB -u %s \n" % str(jd.job_contact)
 
     # if total_cpu_count is not defined, we assume 1
     if jd.total_cpu_count is None:
-        jd.total_cpu_count = 1
+        total_cpu_count = 1
+    else:
+        total_cpu_count = jd.total_cpu_count
 
     lsf_params += "#BSUB -n %s \n" % str(jd.total_cpu_count)
```

Comment:
This causes an error on Summit since '-n' on Summit "Submits a parallel job and specifies the number of tasks in
the job." (source: 'man BSUB').